test/src/test/java/org/corfudb/integration/LogReplicationDynamicStreamIT.java

```diff
@@ -0,0 +1,386 @@
+                IntValue.class,
+                Metadata.class,
+                TableOptions.fromProtoSchema(ValueFieldTagOne.class)
+        );
+
+        Table<StringKey, IntValue, Metadata> mapTagOneStandby = corfuStoreStandby.openTable(
+                NAMESPACE,
+                streamName,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.fromProtoSchema(ValueFieldTagOne.class)
+        );
+
+        // Subscribe the testing stream listener
+        UUID streamId = CorfuRuntime.getStreamID(mapTagOne.getFullyQualifiedTableName());
+        // The plus one is for the clear entry before applying the stream updates
+        int expectedMessageSize = numWrites + 1;
+        CountDownLatch streamingStandbySnapshotCompletion = new CountDownLatch(expectedMessageSize);
+        StreamingStandbyListener listener = new StreamingStandbyListener(streamingStandbySnapshotCompletion,
+                Collections.singleton(streamId));
+        corfuStoreStandby.subscribeListener(listener, NAMESPACE, DefaultLogReplicationConfigAdapter.TAG_ONE);
+
+        writeToMap(mapTagOne, true, 0, numWrites);
+
+        // The stream to tags map is rebuilt upon receiving the replicated data
+        startLogReplicatorServers();
+
+        // Verify snapshot sync is succeed and stream listener received all the changes
+        verifyDataOnStandby(mapTagOneStandby, numWrites);
+
+        streamingStandbySnapshotCompletion.await();
+        assertThat(listener.messages.size()).isEqualTo(expectedMessageSize);
+    }
+
+    /**
+     * This test will verify the streams to stream tags map is correctly rebuilt during
+     * log entry sync.
+     *
+     * (1) Perform a basic snapshot sync and log entry sync, verify the cluster is in log entry sync state
+     * (2) Open a new map on both Source and Sink with TAG_ONE
+     * (3) Initiate a testing stream listener and subscribe to TAG_ONE on Sink
+     * (4) Write some new entries to this new map on Source
+     * (5) Verify new data get replicated and the stream listener received all the updates
+     */
+    @Test
+    public void testStandbyStreamingLogEntrySync() throws Exception {
+        // perform basic snapshot and log entry sync, the cluster should be in log entry sync state now
+        testSnapshotAndLogEntrySync();
+
+        // Verify log replication status is log entry sync
+        verifyInLogEntrySyncState();
+
+        // Open testing map on Source and Sink
+        String streamName = "TableStreaming001";
+
+        Table<StringKey, IntValue, Metadata> mapTagOne = corfuStoreActive.openTable(
+                NAMESPACE,
+                streamName,
+                StringKey.class,
+                IntValue.class,
+                Metadata.class,
+                TableOptions.fromProtoSchema(ValueFieldTagOne.class)
+        );
+
+        Table<StringKey, IntValue, Metadata> mapTagOneStandby = corfuStoreStandby.openTable(
```

Comment:
same here - you can open the map only on the Source cluster for another test

Reply:
Same as this: https://github.com/CorfuDB/CorfuDB/pull/3315#discussion_r975955643

Reply:
here also, you can verify without the listener